test/langtools/tools/javac/DefiniteAssignment/DA_DUConstructors.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * @test /nodynamiccopyright/
+ * @bug 8324873
```

Comment:
just paperwork-ish, I'm not sure we should refer to a valhalla bug and keep the valhalla summary below. I get why you are doing it but I guess this current fix will be pushed before valhalla. So I would just use this current bug ID and summary and once we do a merge of this test in the valhalla repo we can add more information to it

Reply:
No I completely agree - two different repo's. Cut & paste glitch.

Fixed in 4d2d3e83c42.